aspnet/security/app-secrets.rst

```diff
@@ -17,66 +17,36 @@ Environment variables
 
-Apps frequently require secrets, such as a client ID for OAuth. These passwords and other sensitive data should never be added to *config* files inside your project's source tree, as configuration files can be accidentally checked into source control. The secrets manager tool provides a mechanism to store sensitive data for development work outside your project tree. The secrets manager tool is a DNX console application that is used to store secrets used by DNX and ASP.NET applications in the development environment.
+Apps frequently require secrets, such as a client ID for OAuth. These passwords and other sensitive data should never be added to *config* files inside your project's source tree, as configuration files can be accidentally checked into source control. The secret manager tool provides a mechanism to store sensitive data for development work outside your project tree. The secret manager tool is a DNX console application that is used to store secrets used by DNX and ASP.NET applications in the development environment.
 
-Installing the secrets manager tool
+Installing the secret manager tool
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
-- Create a new ASP.NET web app. We will use this to test secrets stored with the secrets manager tool.
+- Create a new ASP.NET web app. We will use this to test secrets stored with the secret manager tool.
 - Open a command prompt and navigate to the project folder (the folder with *config.json*).
 - Set the runtime version using the .Net Version Manager (DNVM). DNVM is a tool that lets you list, install and switch :doc:`DNX </dnx/overview>` versions on your machine. Run the following command:
 
-.. TODO each version, update version # (1.0.0-beta4)
-	note: there are multiple versions in this file
-
 .. code-block:: none
 
-  dnvm use 1.0.0-beta4
-    
-- Install the secrets manager tool using DNU (Microsoft .NET Development Utility). DNU is used to build, package and publish DNX projects.
- 
-.. code-block:: none
- 
-  dnu commands install SecretManager
-
-.. note::
-  With beta4, you will get the following error::
+  dnvm use 1.0.0-beta5
 
-    Unable to locate SecretManager >= 1.0.0-beta4-10173
+The **dnvm** tool is the .NET Version Manager used to update and configure the .NET Execution Environment (DNX). The command ``dnvm use default`` instructs the .NET Version Manager to add the DNX to the ``PATH`` environment variable for the current shell. After running this command the following is displayed::
 
```

Comment:
The command ``dnvm use default`` instructs  --> should be
The command ``dnvm use 1.0.0-beta5 `` instructs .. 

to add the [Beta5 version of] DNX  <-- Add beta5?